docker/build.sh

```diff
@@ -14,7 +14,6 @@ rsync -a \
   --exclude='.idea/' \
   --exclude='docker/' \
   --exclude='cache/' \
-  --exclude='README.md' \
```

Comment:
@bitromortac How do you feel about just excluding README.md from the docker build alltogether? That is, the package generated in the image would have `long_description` fall back to be equal to `description`, as opposed to containing the whole readme. Would reduce image size and make the build a bit cleaner, but if you feel strongly about keeping the build result identical, I'll keep it